main.go

```diff
@@ -118,6 +118,11 @@ func main() {
 			Usage:  "squash the layers at build time",
 			EnvVar: "PLUGIN_SQUASH",
 		},
+		cli.BoolTFlag{
+			Name:   "cache",
+			Usage:  "don't attempt to re-build layers of the image that already exist",
+			EnvVar: "PLUGIN_USE_CACHE",
```

Comment:
note that we cannot use `pull` here. This is a reserved keywork in the yaml and will not get passed to the plugin. So instead we use `pull_image`